README.md

```diff
@@ -265,6 +267,26 @@ There are following known limitations of native OTel logs collection:
   Logs collection with fluentd is still recommended if chart deployed with `autodetect.istio=true`.
 - Journald logs cannot be collected natively by Splunk OTel Collector yet.
 
+### Performance of Splunk Connect for Kubernetes-OpenTelemetry
+
+Some configurations used with Splunk Connect for Kubernetes-OpenTelemetry can have an impact on overall performance of log ingestion. The more receivers, processors, exporters and extensions that are added to any of the pipelines, the greater the performance impact.
+
+Splunk Connect for Kubernetes-OpenTelemetry can exceed the default throughput of HEC. To best address capacity needs, Splunk recommends that you monitor the HEC throughput and back pressure on Splunk Connect for Kubernetes-OpenTelemetry deployments and be prepared to add additional nodes as needed.
```

Comment:
Should cover both Splunk platform and LO receivers? and remove reference to SCK-OTel...  Below is correction for Splunk platform.

"When sending data to Splunk, it is possible to exceed the default throughput of HEC. To best address capacity needs, Splunk recommends that you monitor the HEC throughput and back pressure on Splunk OpenTelemetry Connector for Kubernetes deployments.

<Remedition for this TBD - we need to provide guidance on action to take if this is an issue for the customer>

Reply:
Since there are no performance comparison numbers between the new chart with fluentd vs the new chart with otel logs collection for LO and the impact of more receivers, we should add that post - I'll continue with the correction for the Splunk platform at the moment as a first iteration of this